### PR TITLE
Cosmetic changes

### PR DESCRIPTION
**Pull request by Pastafarianist**
* Fixed lots of typos.
* Files under `scripts/tls-files` no longer match with `.gitignore`.
* Renamed `Pos.Lrc.Class` to `Pos.Lrc.RichmenComponent`.
* Improved descriptions of some modules.
* Removed commented-out code.

### CODE REVIEW

File: ssc/Pos/Ssc/Error/Seed.hs [hunk 1]
```diff
@@ -1,4 +1,4 @@
--- | Error which can arise during seed calculation.
+-- | Possible failures during SSC seed calculation.
```
Comment (neongreen) [inline ssc/Pos/Ssc/Error/Seed.hs]: Shared seed calculation seed calculation :trollface: But okay, it's clearer this way.

File: wallet/src/Pos/Util/BackupPhrase.hs [hunk 2]
```diff
@@ -40,7 +40,7 @@ englishWords = [ "recycle" , "child" , "universe" , "extend" , "edge" , "tourist
                , "color" , "screen" , "birth" , "detect" , "summer" , "palm"
                , "entry" , "swing" , "fit" , "garden" , "trick" , "timber"
                , "toss" , "atom" , "kitten" , "flush" , "master" , "transfer"
-               , "success" , "worry" , "rural" , "silver" , "invest" , "mean "
+               , "success" , "worry" , "rural" , "silver" , "invest" , "mean"
```
Comment (Pastafarianist) [inline wallet/src/Pos/Util/BackupPhrase.hs]: This was confirmed by @adinapoli-iohk as being correct.

Comment (neongreen) [inline wallet/src/Pos/Util/BackupPhrase.hs]: This might break something. Not sure.

Comment (gromakovsky) [reply wallet/src/Pos/Util/BackupPhrase.hs]: Yes, I am also afraid of breakages.

**Comment by neongreen** [comment]
> This was confirmed by @adinapoli-iohk as being correct.

Okay, fine then.

**Comment by adinapoli-iohk** [comment]
> I'm worried about the "mean " thing and would like someone more knowledgeable to confirm that it's alright.

Yep, that was copied & pasted by me when trying to expose a more sensible example for our Swagger V1 API, it won't break anything 😉

File: ssc/Pos/Ssc/Error/Seed.hs (hunk 1 above)
Comment (Pastafarianist) [reply ssc/Pos/Ssc/Error/Seed.hs]: * `shared seed computation seed calculation`, I guess :)

**Pull request merged**